templates/rke2/overlay/opt/corral/rke2/init-agent.sh

```diff
@@ -10,6 +12,21 @@ tls-san:
   - ${CORRAL_api_host}
 EOF
 
+else
+
+mkdir -p /etc/rancher/rke2
+cat > /etc/rancher/rke2/config.yaml <<- EOF
+write-kubeconfig-mode: 644
+server: https://${CORRAL_kube_api_host}:9345
+cni: ${CORRAL_cni}
+system-default-registry: ${CORRAL_registry_fqdn}
+token: ${CORRAL_node_token}
+tls-san:
+  - ${CORRAL_api_host}
+EOF
+
+fi
```

Comment:
Format

Reply:
This is from Izaac's work. Should probably be addressed his PR, and I can rebase to receive the changes.